Scripts/fn_toggleLights.sqf

```diff
@@ -0,0 +1,47 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Muta, TyroneMF
+ * Switches Lights on or off in a defined radius.
+ * Call from init.sqf or Trigger.
+ *
+ * Arguments:
+ * 0: Marker <STRING>
+ * 1: Radius <NUMBER>
```

Comment:
Can't you define the radius with a marker? `(get)markerSize`

Reply:
You can, but it gives you an array with 2 numbers. It only expects one. The simple solution with this is probabbly best as it's just simply placing a marker and selecting how far out you want it to take effect.